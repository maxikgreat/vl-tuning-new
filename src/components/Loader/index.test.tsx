import {render} from '@testing-library/react';

import {Loader} from './index';

describe('Loader component', () => {
	it('should render and match snapshot', () => {
		const {container, asFragment} = render(<Loader />);

		expect(container.querySelector('.lds-roller')).toBeInTheDocument();
		expect(asFragment()).toMatchSnapshot();
	});
});
