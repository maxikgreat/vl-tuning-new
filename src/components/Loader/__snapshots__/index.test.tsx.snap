// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader component should render and match snapshot 1`] = `
<DocumentFragment>
  <div
    class="lds-roller"
  >
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
  </div>
</DocumentFragment>
`;
